### PR TITLE
first commit with relations

### PR DESCRIPTION
**Pull request by kasiaekiert**
Nie wiem czemu, ale ten fragment nie działa 
<td>
            <% if event.user == current_user %>
              <%= link_to 'Edit', edit_event_path(event) %>
            <% end %>
          </td>
          <td>
            <% if event.user == current_user %>
              <%= link_to 'Destroy', event, method: :delete, data: { confirm: 'Are you sure?' } %>
            <% end %>
          </td>

jeśli to zakomentujesz lub usuniesz to wyświetla się events show i members

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Event < ApplicationRecord
   belongs_to :place
-  belongs_to :user
```
Comment (womanonrails) [inline app/models/event.rb]: You should not remove this relation. This is relation which tells us who is the owner of event. You should add new relation `has_and_belongs_to_many :users` which will tell us who is going to this event. Probably name `users` is not the best here. We will take care of it soon. This change is also the reason why your user condition stopped working.

File: app/models/user.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-  has_many :events
```
Comment (womanonrails) [inline app/models/user.rb]: This relation also should stay but we will have problem that two different methods have the same name. This is not good but don't bother this too much now. I will explain you solution soon.

File: app/models/event.rb (hunk 1 above)
Comment (kasiaekiert) [reply app/models/event.rb]: like that:

class Event < ApplicationRecord
  belongs_to :place
  has_and_belongs_to_many :users
  belongs_to :user

  validates :name, name_lenght: { minimum: 5 }
end
??

Comment (womanonrails) [reply app/models/event.rb]: Yes, but you can group all `belongs_to` to one place.

Comment (womanonrails) [reply app/models/event.rb]: Yes, but you can group all `belongs_to` together.

File: app/validators/name_lenght_validator.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class NameLenghtValidator < ActiveModel::EachValidator
```
Comment (womanonrails) [inline app/validators/name_lenght_validator.rb]: Why this validator is here? This is not a part of relation task. It should be on separate branch or on develop. Please remove it from here.

File: app/models/event.rb (hunk 1 above)
Comment (kasiaekiert) [reply app/models/event.rb]: I have tried to do it in one line.... but doesn`t work...

Comment (womanonrails) [reply app/models/event.rb]: No I mean, something like alphabetical order. So you will have `belongs_to` line by line and then the rest.

File: app/models/event.rb [hunk 4]
```diff
@@ -1,6 +1,5 @@
 class Event < ApplicationRecord
-  belongs_to :place
+  belongs_to :place 
   belongs_to :user
-
-  validates :name, name_lenght: { minimum: 5 }
```
Comment (womanonrails) [inline app/models/event.rb]: Why did you remove this validation? If I'm right it should be on develop already.

File: app/views/profile/show.html.erb [hunk 5]
```diff
@@ -7,4 +7,10 @@
   <div class="actions">
     <%= form.submit %>
   </div>
-<% end %>
\ No newline at end of file
+<% end %>
+
+<ul>
```
Comment (womanonrails) [inline app/views/profile/show.html.erb]: Please add also owned events as a second list. Both lists with a headers.
